jobs/mysql-metrics/spec

```diff
@@ -35,6 +38,9 @@ properties:
   mysql-metrics.origin:
     description: "origin tag when metrics are emitted"
     default: p-mysql
+  mysql-metrics.source_id:
+    description: "the source_id when metrics are emitted"
```

Comment:
When we merge this, let's clarify the description on origin and source_id. They represent the same id (the component from where the metrics are emitted). The origin tag is for backwards compatibility with the old dropsonde api, so that the metrics that come out in the new world are exactly the same. The source_id is the new way of communicating this info. Both should be configured and match, if you care about backwards compatibility of metrics.

Or, alternatively, Nitya suggested we could just take one property, origin, and set the source_id to match origin in the config. This might be better

Reply:
Thanks for the review.
Yes, The origin tag is added for backwards compatibility so the messages emitted still have origin tag in it, so that we do not break other old components who still consume this tag.
It's a good idea we can have one property instead, I'd prefer source_id over origin, cause source_id is the new name in the v2 loggregator API spec. We'll push a change for that shortly see if it works better for you.